components/create_comment/create_comment.jsx

```diff
@@ -196,8 +196,6 @@ class CreateComment extends React.PureComponent {
          */
         selectedPostFocussedAt: PropTypes.number.isRequired,
 
-        isMarkdownPreviewEnabled: PropTypes.bool.isRequired,
```

Comment:
Did this logic get moved elsewhere in the past?

Reply:
I assume you found it since you crossed out your comment, but for anyone looking in the future, `isMarkdownPreviewEnabled` is already provided by `TextboxLinks`'s `mapStateToProps`, so this was redundant.